redshift/redshift.go

```diff
@@ -49,68 +87,76 @@ func NewRedshift(host, port, db, user, pwd string, timeout int, s3Info S3Info) (
 }
 
 func (r *Redshift) logAndExec(cmd string) (sql.Result, error) {
-	log.Print("Executing Redshift command: ", cmd)
+	log.Print("Executing Redshift command: ", cmd) // TODO: filter out creds, perhaps
```

Comment:
Agreed to this todo, though I'd have no idea how to remove creds from the cmd, unless we make "logAndExec" take in a command and creds separately. Actually, that's not a bad idea.